change/@fluentui-react-native-button-6e9b526b-44b9-44c1-a7d4-714054ace485.json

```diff
@@ -1,7 +0,0 @@
-{
-  "type": "patch",
```

Comment:
I think this should stay, this package hasn't been updated as part of the bump

Reply:
Ah, I think I'm just gonna redo this again as a whole and not depend on beachball and just bump numbers / delete change files manually..

Reply:
Brought back